src/app/courses/course-item/course-item.component.html

```diff
@@ -1,8 +1,16 @@
-<div class="course">
+<div 
+  class="course"
+  [appFreshCourse]="course.creationDate">
   <div class='course-content'>
-    <div class="course-content-title">{{course.title}}</div>
-    <div class="course-content-duration">{{course.duration}}</div>
-    <div class="course-content-date">{{course.creationDate}}</div>
+    <div class="course-content-title">
+      {{course.title | uppercase}}
+      <fa-icon
+        [icon]="STAR_ICON"
+        [ngClass]="{'course-content-title-star': course.topRated}">
```

Comment:
`Add topRated field to the course item. If true - show star icon, change background color using ngClass.`
Means that if your course is top rated your should not only show a star, but change background color of your course item.

Reply:
changed